### PR TITLE
Add supplementary descriptions in SSN README.

### PR DESCRIPTION
**Pull request by tony2016uestc**

### CODE REVIEW

**Review by dreamerlin — APPROVED** [review]

File: configs/localization/ssn/README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ According to the [Linear Scaling Rule](https://arxiv.org/abs/1706.02677), you ma
 e.g., lr=0.01 for 4 GPUs * 2 video/gpu and lr=0.08 for 16 GPUs * 4 video/gpu.
 2. Since SSN utilizes different structured temporal pyramid pooling methods at training and testing, please refer to [ssn_r50_450e_thumos14_rgb_train](/configs/localization/ssn/ssn_r50_450e_thumos14_rgb_train.py) at training and [ssn_r50_450e_thumos14_rgb_test](/configs/localization/ssn/ssn_r50_450e_thumos14_rgb_test.py) at testing.
 3. We evaluate the action detection performance of SSN, using action proposals of TAG. For more details on data preparation, you can refer to thumos14 TAG proposals in [Data Preparation](/docs/data_preparation.md).
+4. The reference performance is evaluated with `ResNet50` backbone, while the initial performance is evaluated with `BNInception` backbone. The comparison of MMAction2 SSN and the reference SSN are carried out under the same setting and data.
```
Comment (innerlee) [inline configs/localization/ssn/README.md:27]: `initial performance` is not exact

Comment (innerlee) [reply configs/localization/ssn/README.md:27]: 4. The SSN in MMAction2 is evaluated with `ResNet50` backbone, while the reference SSN is evaluated with `BNInception` backbone. Other than backbone, the comparison of MMAction2 SSN and the reference SSN is carried out under the same setting and data.

**Comment by innerlee** [comment]
Okay I'm confused now. What does reference mean? What is its setting?

**Comment by tony2016uestc** [comment]
I ran MMAction SSN with `ResNet50` backbone, and called it as reference SSN. The difference of reference SSN and MMAction SSN lies in backbone.

**Comment by innerlee** [comment]
A: MMAction2, Res50
B: MMAction, Res50
C: MMAction, BNInception

Which one is reference

**Comment by tony2016uestc** [comment]
B. And I compared A with B in the table above.

**Comment by innerlee** [comment]
I see

File: configs/localization/ssn/README.md (hunk 1 above)
Comment (innerlee) [reply configs/localization/ssn/README.md]: @tony2016uestc I updated the description. Hope this version is clearer

Comment (tony2016uestc) [reply configs/localization/ssn/README.md]: OK, thank you very much!

**Review by innerlee — APPROVED** [review]

**Pull request merged**